pyproject.toml

```diff
@@ -0,0 +1,50 @@
+[tool.poetry]
+name = "onionshare"
+version = "2.2"
```

Comment:
Can you add a note to the release documentation (https://github.com/micahflee/onionshare/blob/develop/BUILD.md#changelog-version-and-signed-git-tag) to update the version in this file with each release?

Reply:
Done